.github/CODEOWNERS

```diff
@@ -1072,6 +1072,12 @@
 # ServiceLabel: %%HybridConnectivity %Mgmt
 # ServiceOwners:                                                   @alexmahonic @PriyankaHariharan @apmehrotra
 
+# PRLabel: %Elastic
```

Comment:
Please move this up to the correct location to maintain alphabetical ordering of labels in the management section.   (this area need to be reorganized, as it hasn't been properly maintained)

Reply:
fixed